ex35.py

```diff
@@ -1,76 +1,81 @@
 from sys import exit
+import re
+
+NUMBERS = re.compile('([0-9]+)$')

```

Comment:
I could have done something like "NUMBERS = '([0-9]+)$'", and then on line 10, done:

if re.match(NUMBERS, choice): ....

But I confess that I just prefer to compile regexes and use the .match() method on the regex object instead of "re.match()" from the module.